File: 8kyu/express-matter/express-matter.py
def expression_matter(a: int, b: int, c: int) -> int:
    return max(
      a + b + c,
      a * b * c,
      a * b + c,
      a * (b + c),
      a + b * c,
      (a + b) * c,
    )


print(expression_matter(2, 1, 2))

File: 8kyu/express-matter/express-matter.go
package main

import (
	"fmt"
)

func ExpressionMatter(a int, b int, c int) int {
	arr := []int{a + b + c, a * b * c, a * b + c, a * (b + c), a + b * c, (a + b) * c}
	max :=0

	for _, val := range arr {
		if val > max {
			max = val
		}
	}

	return max
}

func main() {
	fmt.Println(ExpressionMatter(2,1,2))
}